### PR TITLE
CIT-41 #time 4h Implement two API, 1 for  get specific  blog-comment …

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
…and 2 for add commit to the specific blog using ModelViewSet . learned about create and validate method in the serializer

### CODE REVIEW

**Comment by codal-uchoudhary** [comment]
12_march :
-APIs for comment section in the blog is completed using ModelViewSet class.
- APIs for like section is pending

File: Projects/My_website/myWebsite/blog/views.py [hunk 1]
```diff
@@ -26,13 +32,31 @@ def get_queryset(self):
         return data
 
 
-class blogComments(APIView):
-    def get(self, request, id):
-        data = Comments.objects.filter(blog=id)
-        serializer = CommentSerializer(data, many=True)
-        return Response(serializer.data)
+# class to get comments of a specific blog
+class blogComments(viewsets.ModelViewSet):
```
Comment (abhi-codal) [inline Projects/My_website/myWebsite/blog/views.py]: @codal-uchoudhary Class name should always start with caps. this should be `BlogComments` . Please update at all places

File: Projects/My_website/myWebsite/blog/urls.py [hunk 2]
```diff
@@ -7,8 +7,7 @@
 router = routers.DefaultRouter()
 router.register("all-blogs", views.blog)
 router.register("my-blogs", views.myBlog)
+router.register("blog-comments", views.blogComments)
+router.register("add-blog-comment", views.addComment)
```
Comment (abhi-codal) [inline Projects/My_website/myWebsite/blog/urls.py]: @codal-uchoudhary Please avoid using keywords like add, update, etc. in Enpoint.

File: Projects/My_website/myWebsite/blog/views.py [hunk 3]
```diff
@@ -26,13 +32,31 @@ def get_queryset(self):
...
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]
+
+    # def create(self, request):
+    #     data = request.data
+    #     serializer = CommentSerializer(data=data)
+    #     if serializer.is_valid():
+    #         serializer.save()
+    #         return Response(serializer.data)
+    #     return Response(serializer.errors)
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/views.py]: @codal-uchoudhary , pls don't keep commented code. If not needed, remove it.

File: Projects/My_website/myWebsite/blog/views.py [hunk 4]
```diff
@@ -26,13 +32,31 @@ def get_queryset(self):
...
 
 
+# class to get user-specific blogs
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/views.py]: @codal-uchoudhary , pls don't add descriptions like this.

use docstrings instead.

**Comment by codal-uchoudhary** [comment]
Code is Updated as per Comments.

**Review by abhi-codal — APPROVED** [review]

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**